cmake/InstallLayout.cmake

```diff
@@ -377,8 +377,10 @@ set(CPACK_RPM_CLIENTS-VERSIONED_PRE_UNINSTALL_SCRIPT_FILE
 # Configuration for DEB
 ################################################################################
 
-set(CPACK_DEBIAN_CLIENTS-DEB_FILE_NAME "${deb-clients-filename}_amd64.deb")
-set(CPACK_DEBIAN_SERVER-DEB_FILE_NAME "${deb-server-filename}_amd64.deb")
+if (CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64")
+  set(CPACK_DEBIAN_CLIENTS-DEB_FILE_NAME "${deb-clients-filename}_amd64.deb")
```

Comment:
Why the if statement and not just replace `amd64` with the processor name?

Reply:
Is there a case where it returns `amd64`?
I can change it to `not aarch64` for now...

Reply:
Ah, I guess we want to generate aarch64.deb packages too. Let me fix.

Reply:
Other than this, it looks good to me (changing from amd64.deb to x86_64.deb could break things downstream of us, so I think this still needs to be an if ... else).